File: Basic_Algorithms/010_function/function.cpp
#include <iostream>
#include <cmath>

int myFunction(int);

int main(){

  std::cout << myFunction(5) << std::endl;

  return 0;
}

int myFunction(int m){
  int x = (int)std::floor(m / 2);

  for(int i = 0; i < 10; i++)
    x = m + x + i;

  return x;
}

File: Basic_Algorithms/010_function/function.java
// in Java, function is named method instead

class Function{
  static public void main(String[] args){
    System.out.println(myFunction(5));
  }

  static int myFunction(int m){
    int x = (int)Math.floor((float)m / 2);

    for(int i = 0; i < 10; i++)
      x += (m + i);

    return x;
  }
}
